File: extensions/nexus-code-search/src/nexus_code_search/eval/fixtures/csharp_app/code/Zoo.cs
namespace Zoo;

class Animal
{
    public virtual string Describe()
    {
        return "animal";
    }
}

class Lion : Animal
{
    public override string Describe()
    {
        return "lion";
    }

    public void Roar()
    {
    }
}

class ZooKeeper
{
    public Animal Create()
    {
        return new Lion();
    }
}

File: extensions/nexus-code-search/src/nexus_code_search/eval/fixtures/java_app/code/Zoo.java
package com.zoo;

class Animal {
    String describe() {
        return "animal";
    }
}

class Lion extends Animal {
    String describe() {
        return "lion";
    }

    void roar() {
    }
}

class Zoo {
    Animal create() {
        return new Lion();
    }
}
